src/cljs/org/broadinstitute/firecloud_ui/common/style.cljs

```diff
@@ -86,6 +86,13 @@
                              :KhtmlUserSelect "none" :MsUserSelect "none"}} props)
    children])
 
+(defn create-link2 [{:keys [href style onClick text]}]
+  [:a (merge
+        {:href (or href "javascript:;")
+         :style (merge {:textDecoration "none" :color (:button-blue colors)} style)}
+        (when onClick {:onClick onClick}))
+   text])
+
 (defn create-link [onClick & children]
```

Comment:
yup, need to figure out how to consolidate create-link and create-link2, as you said you'll do in a follow-on PR